### ISSUE TITLE
Vim-vixen cannot be installed on Firerox 83

### ISSUE THREAD

**Issue by lruzicka** [ueokande/vim-vixen#940]
### I'm opening this issue because:

- I'll report a bug

### Description

**vim-vixen** plugin cannot be installed on Firefox 83 (Fedora). 

### Failure Information (for bugs)

Any attempt to install the plugin ends with *Plugin could not be installed because it seems to be corrupt.*

#### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Search for the vim-vixen plugin in Add-on Library
2. Try to install it.

#### System configuration

- Operating system: Fedora 33
- Firefox version: 83 
- Vim-Vixen version: 0.31

**Comment by snprajwal** [comment]
Seems like your Firefox has some issues. Please try `dnf reinstall firefox`

**Comment by lruzicka** [comment]
Yeah, there is a bug reported in Fedora (https://bugzilla.redhat.com/show_bug.cgi?id=1908018) in the meantime for this Firefox behaviour. I have solved the situation by allowing unsigned add-ons and now Vim Vixen works again.
